packages/core/src/statistics/Statistics.ts

```diff
@@ -2,29 +2,35 @@ import { Logger } from '@stryker-mutator/api/logging';
 import { commonTokens, tokens } from '@stryker-mutator/api/plugin';
 import { HttpClient } from 'typed-rest-client/HttpClient';
 
-const AZURE_URL = 'https://localstrykertryout.azurewebsites.net/api/HttpTrigger?code=4GWK/KLC6JRlec96851wMgMsD2JkVePiaALWw6lKv4R3RZiKf0xp0w==';
+import { JsonLoader } from './JsonLoader';
+
+const AZURE_URL = 'https://strykerstatistics.azurewebsites.net/api/ReceiveStatistics?code=jVZfGmoB6ofRPa/yPdN/mAOCd6ia67XQkTmLaGWCzlxO5a32PlLj6A==';
 
 export class Statistics {
-  public static inject = tokens(commonTokens.logger, 'httpClient');
-  public statistics: any = {};
+  public static inject = tokens(commonTokens.logger, 'httpClient', 'testRunner');
+  public statistics: Record<string, any> = {};
 
-  constructor(private readonly log: Logger, private readonly httpStatisticsClient: HttpClient) {
+  constructor(private readonly log: Logger, private readonly httpStatisticsClient: HttpClient, private readonly testRunner: string) {
     this.statistics.implementation = 'Stryker';
-    this.statistics.version = require('../../package.json').version;
-    this.statistics.testRunner = require('../../../../stryker.parent.conf').testRunner;
+    this.statistics.testRunner = this.testRunner;
   }
 
   public addStatistic(name: string, value: any) {
     this.statistics[name] = value;
   }
 
+  private setGenericData() {
+    this.statistics.version = JsonLoader.loadFile('../../package.json').version;
+  }
+
   public sendStatistics(): Promise<void> {
+    this.setGenericData();
     this.log.info(`Sending anonymous statistics to ${AZURE_URL}`);
     const statisticsData = JSON.stringify(this.statistics);
-    this.log.info(statisticsData); // DEBUG
+    this.log.info(statisticsData);
     return this.httpStatisticsClient
       .post(AZURE_URL, statisticsData, {
-        ['Content-Type']: 'application/json'
+        ['Content-Type']: 'text/plain'
```

Comment:
Waarom is dit veranderd naar `text/plain`? Zelfs met `JSON.stringify` is het nog steeds JSON wat je opstuurt

Reply:
Dit is een foutje. Ik was aan het testen of in Azure Functions de gegevens automatisch als JSON object wordt geparsed. Ik heb er weer application/json van gemaakt.